### ISSUE TITLE
Record gpx file?

### ISSUE THREAD

**Issue by Offerel** [gege13007/Amazfit-Stratos-GPS-locator#6]
Im searching for an app, which records the actual position on tab of the watch screen and export this later as a gpx file. The use case is it, to safe battery as much as i can, but record the gps position on a tab. This would be useful for taking pictures with a DSLR without a GPS chip and match later foto and gps with the gpx file. Is that possible with this app?

**Comment by Uwe-B** [comment]
Yeah that would be nice. But another app of the same developer seems to do that, though I miss the opportunity to set it up (intervall and timeframe) for my purposes:
https://github.com/gege13007/Amazfit-GPS-SlowLog

**Comment by Offerel** [comment]
I will test this one out. Currently i have no idea how i can sideload a apk file to my watch, but i will do my best and report.

**Comment by Uwe-B** [comment]
Sideloading is easy via adb. There are some tools to make it easier, most of them can be found here:
https://forum.xda-developers.com/f/amazfit.6606/
